style.css

```diff
@@ -60,8 +64,11 @@ background-image: url('images/header.jpg');
   padding: 14px 10px;
   text-decoration: none;
   font-size: 20px;
-  position: relative;
+  word-spacing: 10px;
+  position: absolute;
```

Comment:
This abs positioning seems to be missing relative pos on parent container.

I think restructuring the html to create better containers and using flexbox would be better than all the abs positioning going on here.